<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="360pc011" name="com.edureka.referralProcess.VerifyReferringFlow" tests="4" failures="0" timestamp="3 Feb 2017 12:13:48 GMT" time="406.182" errors="2">
  <testcase name="test_003VerifyEarningCredtiPoints" time="22.591" classname="com.edureka.referralProcess.VerifyReferringFlow">
    <error type="org.openqa.selenium.NoSuchElementException" message="Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//span[@class=&#039;webinar-profile-name&#039;]&quot;}
Command duration or timeout: 17 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.48.1&#039;, revision: &#039;d80083d&#039;, time: &#039;2015-10-08 21:11:00&#039;
System info: host: &#039;360pc011&#039;, ip: &#039;127.0.1.1&#039;, os.name: &#039;Linux&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;4.2.0-27-generic&#039;, java.version: &#039;1.7.0_111&#039;
*** Element info: {Using=xpath, value=//span[@class=&#039;webinar-profile-name&#039;]}
Session ID: 32fdd595-b4cd-48f9-8b6e-f9e7cefd2734
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=41.0.2}]">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//span[@class='webinar-profile-name']"}
Command duration or timeout: 17 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: '360pc011', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.2.0-27-generic', java.version: '1.7.0_111'
*** Element info: {Using=xpath, value=//span[@class='webinar-profile-name']}
Session ID: 32fdd595-b4cd-48f9-8b6e-f9e7cefd2734
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=41.0.2}]
	at sun.reflect.GeneratedConstructorAccessor24.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at com.edureka.pages.UserHomePage.verifyUserPage(UserHomePage.java:36)
	at com.edureka.referralProcess.VerifyReferringFlow.test_003VerifyEarningCredtiPoints(VerifyReferringFlow.java:377)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//span[@class='webinar-profile-name']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.1', revision: 'd80083d', time: '2015-10-08 21:11:00'
System info: host: '360pc011', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.2.0-27-generic', java.version: '1.7.0_111'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///tmp/anonymous3053520068614860405webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10659)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///tmp/anonymous3053520068614860405webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10668)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///tmp/anonymous3053520068614860405webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12534)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///tmp/anonymous3053520068614860405webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12539)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///tmp/anonymous3053520068614860405webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12481)
]]>
    </error>
  </testcase> <!-- test_003VerifyEarningCredtiPoints -->
  <testcase name="test_005VrifyRedemptionOFCreditPoints" time="0.000" classname="com.edureka.referralProcess.VerifyReferringFlow">
    <error type="java.lang.Throwable" message="Method VerifyReferringFlow.test_005VrifyRedemptionOFCreditPoints()[pri:0, instance:com.edureka.referralProcess.VerifyReferringFlow@691574db] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method VerifyReferringFlow.test_005VrifyRedemptionOFCreditPoints()[pri:0, instance:com.edureka.referralProcess.VerifyReferringFlow@691574db] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1040)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- test_005VrifyRedemptionOFCreditPoints -->
  <testcase name="test_002RefferralActivities" time="258.133" classname="com.edureka.referralProcess.VerifyReferringFlow"/>
  <testcase name="test_001VerifyRefferringFlow" time="125.458" classname="com.edureka.referralProcess.VerifyReferringFlow"/>
</testsuite> <!-- com.edureka.referralProcess.VerifyReferringFlow -->
